### PR TITLE
Javascript operators and expressions

### PR DESCRIPTION
**Pull request by EuichangPark**
javascript-operators-and-expressions

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+var firstName = 'Euichang';
+var lastName = 'Park';
+var fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName: ', fullName);
+console.log('type of fullName: ', typeof fullName);
+
+var pH = 6;
+var isAcidic = false;
+if (pH < 7) {
```
Comment (Zhiwutian) [inline javascript-operators-and-expressions/main.js:32]: you don't want any `if` statements in here, you are just saving the result of the expression into a variable and then logging it out.

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  console.log('Value of isAcidic: ', isAcidic);
+  console.log('type of isAcidic: ', typeof isAcidic);
+}
+
+var headCount = 7;
+var isSparta = false;
+if (headCount === 300) {
```
Comment (Zhiwutian) [inline javascript-operators-and-expressions/main.js:40]: same as above!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
No if statements for this one, just saving the result of the expressions into a variable.

**Comment by EuichangPark** [comment]
javascript-operators-and-expressions

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**